### ISSUE TITLE
Port to GemStone 3.7.0

### ISSUE THREAD

**Issue by dalehenrich** [SeasideSt/Seaside#1355]
WARemoteDebuggingWalkback>>resumeContinuation sends the message #_continue to GsProcess instance after #resumeContinuation is sent and conditionally sends second #_continue message ... 

In 3.7.0 we updated the GsProcess implementation to improve it's muti-threaded behavior and in the process the GsProcess>>_continue method was removed ...

I'm not sure how much WARemoteDebuggingWalkback is used in the wild, but at a minimum we shouldn't fail the Seaside tests because of this "sent but not implemented" message ...

In 3.7.0 we have also done significant work in using topaz for remote debugging with the [DEBUGGEM topaz command](https://downloads.gemtalksystems.com/docs/GemStone64/3.6.x/GS64-Topaz-3.6/3-Commands.htm#pgfId-1181330) and I would be inclined to replace the WARemoteDebuggingWalkback functionality with support for DEBUGGEM ... 

In a nutshell by using `System waitForDebug` a process id and unique token are written to the gem's log file and the process is suspended until a topaz session running on the same machine attaches to the process ... once attached you can debug the process using a topaz debugger and subsequently continue execution, kill the gem, or  detach and leave process suspended ...

**Comment by dalehenrich** [comment]
Looks like JQJsonFunctionalTest>>seasidePackagesOn: sends #array: that is not implemented in GemStone ... did not show up as a test failure during internal battery tests runs and I found it while characterizing #_continue issue

**Comment by dalehenrich** [comment]
ignore the read 'x' ... [test for b3831fb have passed](https://github.com/SeasideSt/Seaside/actions/runs/4256204853)

**Comment by jbrichau** [comment]
Or actually... the issue with the `WARemoteDebuggingWalkback` not working anymore starting GS 3.7 should be handled. Ok, leave this one open and I will create a separate issue for it before closing this one.
